EndlessLauncher/App.xaml.cs

```diff
@@ -47,11 +48,39 @@ protected override void OnStartup(StartupEventArgs e)
                 Debug.SetDebugSimulatedError(errorCode);
             }
 
-            mutex = new Mutex(true, "{5E80890D-A4E6-4B8C-B123-FFD89450547F}", out bool isNew);
-            if (!isNew)
+            if (e.Args.Length > 0)
             {
-                Utils.ActivateWindow(null, "MainWindow");
-                Shutdown();
+                string command = String.Join(" ", e.Args);
+
+                switch (command)
+                {
+                    case "/Kiwix":
+                        LauncherShortcuts.OpenKiwix();
+                        Shutdown();
+                        break;
+                    case "/Kolibri":
+                        LauncherShortcuts.OpenKolibri();
+                        Shutdown();
+                        break;
+                    case "/Readme":
```

Comment:
I don't think we need this, they could just double click the PDF file, but it's not harmful.